dev/source/docs/ardupilot-discord-server.rst

```diff
@@ -0,0 +1,33 @@
+.. _ardupilot-discord-server:
+
+========================
+ArduPilot Discord Server
+========================
+
+The ArduPilot dev team uses `Discord <https://ardupilot.org/discord>`__ for real-time voice chat.  We have a "General Discussion" channel for chatting about ArduPilot development at any time. How many developers are connected depends a lot on timezones.  Don't be shy about your English language ability, we have developers from all over the world.
```

Comment:
Is the main discord channel only for development?  It seems to just be a huge chat..

Reply:
I think we decided this was OK as-is via... Discord :-)